### ISSUE TITLE
Understanding SC 2.1.2 - broken link

### ISSUE THREAD

**Issue by jiri-thiemel-te** [w3c/wcag#3573]
On the page https://www.w3.org/WAI/WCAG21/Understanding/no-keyboard-trap.html in the SC Note there's a link called
"Conformance Requirement 5: Non-Interference"
which is broken.

**Comment by patrickhlauke** [comment]
> I believe this is intended to point to https://www.w3.org/TR/WCAG21/#cc5

well yes, but this is another one of those technical problems we have with the way the spec/understanding docs are built. the note in the SC text has a relative link, which then works in the main WCAG spec. however, when the SC is pulled into the understanding document, the relative link doesn't work anymore. I believe the solution here is technical / to be fixed at source in the relevant scripting/xslt that builds understanding docs. /cc @iadawn

**Comment by jiri-thiemel-te** [comment]
Shouldn't it rather point to Understanding Conformance page since the link is on Understanding SC 2.1.2 page?
https://www.w3.org/WAI/WCAG21/Understanding/conformance#conf-req5

**Comment by patrickhlauke** [comment]
@alastc this is probably another one of those "how do we make references in light of how WCAG documents get built"

**Comment by bruce-usab** [comment]
Ping to @iadawn via @avkuo and the wcag2-issues tf.  Please advise.

**Comment by iadawn** [comment]
This is related to #3611. I will need to rebuild all the 2.1 docs with the updated XSLT file

**Comment by patrickhlauke** [comment]
@kfranqueiro this may be of interest to you as well (as it relates to build process etc)

**Comment by kfranqueiro** [comment]
This is resolved now that WCAG 2.1 Techniques and Understanding Docs have been republished via the new build system, made possible by https://github.com/w3c/wcag/pull/4007.

> Shouldn't it rather point to Understanding Conformance page since the link is on Understanding SC 2.1.2 page? https://www.w3.org/WAI/WCAG21/Understanding/conformance#conf-req5

The contents within the SC box are sourced from the `guidelines` folder contents as-is (i.e. the same content that appears directly in the TR); adjusting links to instead point to Understanding counterparts would require another layer of logic (and might not be entirely straightforward in cases such as this where the URI fragment changes.)
